### ISSUE TITLE
Invalid feed generated due to lack of entity encoding

### ISSUE THREAD

**Issue by cmpilato** [albertobeta/PodcastGenerator#182]
My podcast uses the iTunes category "Religion & Spirituality:Christianity".  PG3.0 is failing to entity-encode the ampersand when generating `feed.xml`, which results in this content of that file:

```
[...]
<itunes:category text="Religion & Spirituality:Christianity"></itunes:category>
[...]
```

Such XML fails to pass parser validation.

**Comment by cmpilato** [comment]
I'm not sure if the right fix is to use `htmlspecialchars()` or CDATA blocks.  But I suspect there are other fields (title, description, copyright, etc.) which suffer from the same encoding bug.

**Comment by emilengler** [comment]
`htmlspecialchars()` will do!
Thanks for the report

**Comment by emilengler** [comment]
Fixed in 135c02b

**Comment by emilengler** [comment]
Included in 3.0.1
